tock/hours/admin.py

```diff
@@ -24,6 +24,22 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class EmployeeListFilter(admin.SimpleListFilter):
```

Comment:
Do we need this? Doesn't look like it's being used anywhere.

Reply:
not needed, removing...